README.md

```diff
@@ -218,6 +218,16 @@ Linux](https://en.wikipedia.org/wiki/Windows_Subsystem_for_Linux) on Windows.
 Please see the sections below for instructions on setting up a particular editor
 or IDE.
 
+### IntelliJ IDEA
+
+If you choose to use [IntelliJ IDEA](https://www.jetbrains.com/idea/) as your
+IDE when working on Skara you can simply open the root folder and the project
+should be automatically imported. You will need to configure a Platform SDK that
+is JDK 14 or above. Either set this up manually, or [build](#building) once from
```

Comment:
We are using JDK 15 now :smile:

Reply:
Sure, but the `Building` section still says that 14 or later is needed, and I guess that is still actually true. :)

Reply:
Ok, sure, I agree.